packages/core/test/insured-bridge/BridgePool.js

```diff
@@ -1037,6 +1278,19 @@ describe("BridgePool", () => {
 
       const relayerBalanceBefore = await l1Token.methods.balanceOf(relayer).call();
       const instantRelayerBalanceBefore = await l1Token.methods.balanceOf(instantRelayer).call();
+
+      // Settle OO request.
+      const proposeEvent = (await optimisticOracle.getPastEvents("ProposePrice", { fromBlock: 0 }))[0];
```

Comment:
for this and all following calls like this: do you need to provide the `fromBlock: 0`? does it not default to this?

Reply:
Nope it doesn't default to this like it used to with truffle

Reply:
o ok. pitty